### ISSUE TITLE
[Bug]: ==> ERROR: Failure while downloading from AUR (Arch Linux)

### ISSUE THREAD

**Issue by dnuser** [FreeTubeApp/FreeTube#6928]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

Hello,

since the last update of Freetube from the AUR with the yay- package manager I get this error: 
==> ERROR: Failure while downloading https://github.com/FreeTubeApp/FreeTube/archive/v0.23.2-beta.tar.gz
    Aborting...
 -> error making: freetube-exit status 1
 -> Failed to install the following packages. Manual intervention is required:
freetube - exit status 1.

I remember that I the first time I tried to update it it made some progress but my internet connection broke. I am unsure if it hast something to do with this issue.

![Image](https://github.com/user-attachments/assets/5ba9b74b-59d9-430e-879d-3d6acfdcbd01)

Any idea what I do wrong or I can do right?

Thanks for help.

Regards

### Expected Behavior

Update/Download the package.

### Issue Labels

accessibility issue

### FreeTube Version

0.23.2

### Operating System Version

Arch Linux

### Installation Method

other

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
AUR is listed under https://github.com/FreeTubeApp/FreeTube?tab=readme-ov-file#unofficial-downloads

> These builds are maintained by the community. While they should be safe, download at your own risk. There may be issues with using these versus the official builds. Any issues specific with these builds should be sent to their respective maintainer. Make sure u always try an [official download](https://github.com/freetubeapp/freetube/#official-downloads) before reporting your issue to us!

**Comment by absidue** [comment]
As mentioned in the README the AUR packages are all unofficial and maintained by the community and any issues with them should be directed at their respective maintainers. Please try using one of the official builds from the FreeTube website https://freetubeapp.io

> These builds are maintained by the community. While they should be safe, download at your own risk. There may be issues with using these versus the official builds. Any issues specific with these builds should be sent to their respective maintainer. Make sure u always try an official download before reporting your issue to us!

**Comment by dnuser** [comment]
Thanks for all the help! I just had to install freetube-bin instead of the freetube package. I missed that. My bad. Somebody gave me that hint thanks for that too.
Thank you all for your kind support.
